### ISSUE TITLE
Resident Evil 2 graphical issue and no sound

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#339]
Originally reported on Google Code with ID 340

```
Describe your system:
 - Mac OS X 10.6.2
- Machine type (32-bit or 64-bit):
 - Mupen64Plus version: 1.5
 - Plugins used: Glide64, glN64 and Rice Video plugin 1.5

For visual artifacts, give:
 - game name and CRC/MD5 hash: Resident Evil 2 (U) v1.1
 - description: artifacts, black screen,

The problem:

Rice and glN64: black screen when starting the rom. Ingame: black screen and artifacts,
no 
sound
glide64: menu works kind of fine. ingame: only player visible. black background and
artifacts, no 
sound
```

Reported by `a.janjic.nlplanet` on 2010-02-24 19:28:01

**Comment by richard42** [comment]
```
I experience the same issue with Mupen64Plus version 1.99.4 on Arch Linux.
```

Reported by `diego.viola` on 2010-12-26 03:40:59

**Comment by richard42** [comment]
```
I get these messages:

Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0
Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0

```

Reported by `diego.viola` on 2010-12-26 03:41:30

**Comment by richard42** [comment]
```
RSP Warning: unknown task:  type:1  sum:2ad18  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:2ad18  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:2ad18  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:1e4e5  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:1e599  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:ccd8  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:1e51f  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:cc5e  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:1e578  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:cca7  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:1e61d  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:cd5c  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:1e61c  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:cd6b  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:1e5b4  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:ccf3  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:1e61c  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:cd6b  PC:b611a5a0
RSP Warning: unknown task:  type:1  sum:1e62d  PC:b611a5a0

```

Reported by `diego.viola` on 2010-12-26 03:41:48

**Comment by richard42** [comment]
```
grrrr, I get the same errors. Not my idea of having fun on a classic game.
```

Reported by `kno0001` on 2011-01-27 03:17:36

**Comment by richard42** [comment]
```
Still same issue on mupen64plus 2.0.0

Video Warning: Unimplemented: write into Z buffer.  Was mostly commented out in Rice
Video 6.1.0

archlinux x86_64
```

Reported by `diego.viola` on 2013-12-12 19:43:19

**Comment by richard42** [comment]
```
The cutscenes will not work but the rest should be ok

Please use the following --rsp:

 * mupen64plus-rsp-cxd4-hlevideo

And following --gfx:

 * mupen64plus-video-glide64mk2

And make sure you change CounterPerOp in [Core] (mupen64plus.cfg) to 1
```

Reported by `sven@narfation.org` on 2013-12-13 23:40:21

**Comment by richard42** [comment]
```
Just to make it clear: The cut scene problem is a regression in glide64mk2 and I will
check it when I have more time
```

Reported by `sven@narfation.org` on 2013-12-14 12:17:50

**Comment by richard42** [comment]
```
You can use the following patch as quick workaround for the missing cutscenes. But the
mupen64plus OSD will be broken
```

Reported by `sven@narfation.org` on 2013-12-15 10:47:57

<hr>
- _Attachment: [glide64mk2_war_biohazard_cutscenes.patch](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-340/comment-10/glide64mk2_war_biohazard_cutscenes.patch)_

**Comment by richard42** [comment]
```
I sent a correct patch to Richard with the next pull request
```

Reported by `sven@narfation.org` on 2013-12-15 11:16:41

<hr>
- _Attachment: [shader_reset.patch](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-340/comment-11/shader_reset.patch)_

**Comment by richard42** [comment]
```
Just as reminder: There is no mupen64plus-rsp-cxd4-hlevideo anymore. Now you have to
use mupen64plus-rsp-cxd4 with rsp-cxd4[DisplayListToGraphicsPlugin]=True
```

Reported by `sven@narfation.org` on 2013-12-17 17:57:13

**Comment by richard42** [comment]
```
The sound problem with mupen64plus-rsp-hle seems to be the same as described in issue
#99
```

Reported by `sven@narfation.org` on 2013-12-17 17:58:00

**Comment by richard42** [comment]
```
The sound issue is a duplicate of issue #99
```

Reported by `sven@narfation.org` on 2013-12-18 12:10:32
- **Status changed**: `Duplicate`
- **Merged into**: #99
